EveripediaNetwork/ep-ui
# Feedback modal

# Add Feedback Modal After Reset Button Click

_Adds a feedback modal that appears after the reset button is clicked in the DraftModal component._

## How should this be tested?

1. Click the reset button.
2. Verify that the feedback modal appears confirming the reset action.

## Linked issues

closes https://github.com/EveripediaNetwork/issues/issues/3052
